{
    "findings": [
        {
            "text": "The vulnerable class django.http.FileResponse is instantiated with the attribute as_attachment set to True, The vulnerable class django.http.FileResponse is instantiated with external input as the *args argument when filename argument is not provided",
            "evidence": [
                {
                    "cell_type": "values_only",
                    "text": "",
                    "rows": [
                        [
                            "gadgets_scanners/applicable/tests/data/python/CVE_2022_36359/applicable/con2.py",
                            "FileResponse(open(filepath, \"rb\"), as_attachment=True)",
                            "8",
                            "The vulnerable class django.http.FileResponse is instantiated with the attribute as_attachment set to True"
                        ],
                        [
                            "gadgets_scanners/applicable/tests/data/python/CVE_2022_36359/applicable/con2.py",
                            "FileResponse(open(filepath, \"rb\"), as_attachment=True)",
                            "8",
                            "The vulnerable class django.http.FileResponse is instantiated with external input as the *args argument when filename argument is not provided"
                        ]
                    ],
                    "column_names": [
                        "Path",
                        "Evidence",
                        "Line Number",
                        "Issue Found"
                    ]
                }
            ],
            "conclusion": "negative"
        }
    ],
    "conclusion": "negative"
}